### PR TITLE
feat: add disclaimer for non-native wallets

### PR DESCRIPTION
**Pull request by luizstacio**
Add disclaimer before accpeting connection with non native wallets. Use a hardcode list.

closes: #266

### CODE REVIEW

**Review by helciofranco — DISMISSED** [review]

**Review by arthurgeron — DISMISSED** [review]

File: packages/react/src/config.ts [hunk 1]
```diff
@@ -1,2 +1,8 @@
 export const CONNECTOR_KEY = 'fuel-current-connector';
 export const BRIDGE_URL = 'https://app.fuel.network/bridge?from=eth&to=fuel';
+export const NATIVE_CONNECTORS = [
```
Comment (arthurgeron) [inline packages/react/src/config.ts:3]: I'd be cleaner to use `.external` from the Fuel SDK version

Comment (luizstacio) [reply packages/react/src/config.ts:3]: Yes, this is just a implementation before we don't have it.

File: packages/react/src/config.ts [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+  'Fuel Wallet',
+  'Fuel Wallet Development',
+  'Fuelet Wallet',
+  'Bako Safe',
```
Comment (arthurgeron) [inline packages/react/src/config.ts:7]: Missing burner wallet

Comment (luizstacio) [reply packages/react/src/config.ts:7]: As burner is dosen't have a review tx page, I think we should keep as external

**Review by arthurgeron — APPROVED** [review]

**Pull request merged**